.github/CODEOWNERS

```diff
@@ -24,25 +24,25 @@ CHANGELOG*
 /filebeat/input/syslog/ @elastic/security-external-integrations
 /filebeat/input/winlog/ @elastic/security-external-integrations
 /filebeat/module/ @elastic/integrations
-/filebeat/module/apache @elastic/integrations
+/filebeat/module/apache @elastic/obs-infraobs-integrations
 /filebeat/module/auditd @elastic/security-external-integrations
 /filebeat/module/elasticsearch/ @elastic/infra-monitoring-ui
-/filebeat/module/haproxy @elastic/integrations
+/filebeat/module/haproxy @elastic/obs-infraobs-integrations
 /filebeat/module/icinga @elastic/integrations
-/filebeat/module/iis @elastic/integrations
-/filebeat/module/kafka @elastic/integrations
+/filebeat/module/iis @elastic/obs-infraobs-integrations
+/filebeat/module/kafka @elastic/obs-infraobs-integrations
 /filebeat/module/kibana @elastic/integrations
 /filebeat/module/kibana/ @elastic/infra-monitoring-ui
 /filebeat/module/logstash @elastic/integrations
 /filebeat/module/logstash/ @elastic/infra-monitoring-ui
```

Comment:
What's the reason for these two being different? Should they both be infra-monitoring-ui? Same above for Kibana

Reply:
probably we should remove the one without the final slash

Reply:
updated two entries as part of commit: https://github.com/elastic/beats/pull/35545/commits/cc966fc978300158b35120a7cbd7b1ba9a8a7810